lib/main_screen.dart

```diff
@@ -37,6 +38,25 @@ class MainScreenState extends State<MainScreen> {
         ),
         title: Text("EventHub"),
         actions: <Widget>[
+          Container(
+            child: StreamBuilder(
+              stream: themeSwitch.themeModeStream,
+              initialData: ThemeMode.system,
+              builder:
+                  (BuildContext context, AsyncSnapshot<ThemeMode> snapshot) {
+                return Switch(
```

Comment:
I think instead of a switch, it's better to use a Sun/Moon icon here because otherwise, the user wouldn;t know the purpose of this switch
You can show a Sun when the current theme is light, and Moon when the current theme is dark